packages/lesswrong/components/posts/TableOfContents/ToCColumn.tsx

```diff
@@ -116,7 +116,7 @@ export const ToCColumn = ({tableOfContents, header, welcomeBox, children, classe
   welcomeBox: React.ReactNode|null
 }) => {
   return (
-    <div className={classNames(classes.root, {[classes.tocActivated]: !!tableOfContents})}>
+    <div className={classNames(classes.root, {[classes.tocActivated]: !!tableOfContents || !!welcomeBox})}>
```

Comment:
I noticed this is causing the layout to be slightly off for posts without a ToC between the `md` and `sm` breakpoints. This is because the grid from the `tocActivated` class is still applied there, but the welcome box is hidden (but still exists). I think this only affects LW since you're the only ones with a welcome box.

<img width="989" alt="Screen Shot 2022-07-21 at 5 40 36 PM" src="https://user-images.githubusercontent.com/9057804/180320148-c30a32d1-7fe0-4794-bed6-fc66d59dd68b.png">

Reply:
There's additionally (or related) a thing whereby the presence or absence of ToC slightly affects the positioning of the central column, this was already true but it's consistent within a page, so seems ok